### PR TITLE
l10n: Correct spelling

### PR DESCRIPTION
**Pull request by Valdnet**
Signed-off-by: Valdnet <47037905+Valdnet@users.noreply.github.com>

### CODE REVIEW

**Review by skjnldsv — APPROVED** [review]

**Review by come-nc — APPROVED** [review]

File: apps/encryption/templates/altmail.php [hunk 1]
```diff
@@ -2,7 +2,7 @@
 /** @var OC_Theme $theme */
 /** @var array $_ */
 
-print_unescaped($l->t("Hey there,\n\nthe admin enabled server-side-encryption. Your files were encrypted using the password \"%s\".\n\nPlease login to the web interface, go to the section \"basic encryption module\" of your personal settings and update your encryption password by entering this password into the \"old log-in password\" field and your current login-password.\n\n", [$_['password']]));
+print_unescaped($l->t("Hey there,\n\nThe administrator enabled server-side-encryption. Your files were encrypted using the password \"%s\".\n\nPlease login to the web interface, go to the section \"Basic encryption module\" of your personal settings and update your encryption password by entering this password into the \"Old log-in password\" field and your current login-password.\n\n", [$_['password']]));
```
Comment (rakekniven) [inline apps/encryption/templates/altmail.php:5]: admin -> administration

File: apps/encryption/templates/mail.php [hunk 2]
```diff
@@ -15,7 +15,7 @@
 					<td width="20px">&nbsp;</td>
 					<td style="font-weight:normal; font-size:0.8em; line-height:1.2em; font-family:verdana,'arial',sans;">
 						<?php
-						print_unescaped($l->t('Hey there,<br><br>the admin enabled server-side-encryption. Your files were encrypted using the password <strong>%s</strong>.<br><br>Please login to the web interface, go to the section "basic encryption module" of your personal settings and update your encryption password by entering this password into the "old log-in password" field and your current login-password.<br><br>', [$_['password']]));
+						print_unescaped($l->t('Hey there,<br><br>The administrator enabled server-side-encryption. Your files were encrypted using the password <strong>%s</strong>.<br><br>Please login to the web interface, go to the section "Basic encryption module" of your personal settings and update your encryption password by entering this password into the "Old log-in password" field and your current login-password.<br><br>', [$_['password']]));
```
Comment (rakekniven) [inline apps/encryption/templates/mail.php:18]: admin -> administration

**Review by rakekniven — CHANGES_REQUESTED** [review]

File: apps/encryption/templates/mail.php (hunk 2 above)
Comment (rakekniven) [reply apps/encryption/templates/mail.php:18]: Keep it neutral like we do in other new strings.

File: apps/encryption/templates/altmail.php (hunk 1 above)
Comment (rakekniven) [reply apps/encryption/templates/altmail.php:5]: Keep it neutral like we do in other new strings.

Comment (Valdnet) [reply apps/encryption/templates/altmail.php]: > admin -> administration

I was wondering about that too, and found it to be a person as administrator.

> Keep it neutral like we do in other new strings.

Done 😉.

**Review by rakekniven — APPROVED** [review]

File: apps/encryption/templates/mail.php (hunk 2 above)
Comment (Valdnet) [reply apps/encryption/templates/mail.php]: Done 😉.

**Comment by rakekniven** [comment]
Do we want to backport such changeds to latest supported NC versions?

**Comment by Valdnet** [comment]
@szaimen This text string still exists unchanged in Transifex. Can you check why?

**Comment by szaimen** [comment]
Sorry no idea

**Comment by szaimen** [comment]
doesnt look like it...

**Pull request merged**